src/app.js

```diff
@@ -1,29 +1,26 @@
-//Importamos configuracion de Express
-import app from './config/express.js';
-
-//Importamos rutas
-import routes from './routes/index.route.js';
-
-import mongoose from 'mongoose';
-
+//Para usar require, hay que importar esto
 import { createRequire } from 'module';
-
 const require = createRequire(import.meta.url);
-
 require('dotenv').config();
 
+//Importamos configuracion de del servidor
+import app from './config/server.js';
+
+// BD Mongo
+import mongoose from 'mongoose';
+
+/** DB connection */
 mongoose
-  .connect('mongodb://mongo/mydatabase', {
+  .connect('mongodb://localhost/data-base', {
     useNewUrlParser: true,
     useUnifiedTopology: true,
   })
-  .then((db) => console.log('Db is connected to', db.connection.host))
+  .then((db) => console.log('Mongo is connected to', db.connection.host))
   .catch((err) =>
     console.log('Error en la conexion de la base de datos: ', err)
```

Comment:
I'd recommend you using a logging library to log messages (whether they are errors, info o warnings). Also, stick to one language (English preferably) on your messages